README.md

```diff
@@ -14,6 +14,8 @@ The Istio Usage Collector is a Go implementation of the [`gather-cluster-info.sh
 
 This data is collected into a JSON or YAML file that can be used for further analysis through our detailed migration estimator tool.
 
+Istio data gathered and reported is based on automatic sidecar injection defined within Istio's MutatingAdmissionWebhooks.
```

Comment:
note we're not running these in ci or do checks. these are solely meant for local runs due to varying user computers -- I'm basically avoiding needing to run these on a runner multiple times to get a baseline and all that to get this out sooner.